tests/api/v1/common/testAssociations.js

```diff
@@ -105,7 +105,7 @@ function testAssociations(path, associations, joiSchema, conf) {
       .expect(constants.httpStatus.OK)
       .expect((res) => {
         expect(res.body).to.be.an('object');
-        expect(res.body).to.have.keys('id', 'name', assoc, 'apiLinks');
+        expect(res.body).to.contains.all.keys('id', 'name', assoc, 'apiLinks');
```

Comment:
Reason for using "all" here but not above?

Reply:
code complete mistake.